resources/migrations/000_migrations.yaml

```diff
@@ -15905,6 +15905,40 @@ databaseChangeLog:
         - sql:
             sql: drop view if exists v_view_log;
 
+  - changeSet:
+      id: v48.00-047
+      author: noahmoss
+      comment: Drop foreign key on recent_views so that it can be recreated with onDelete policy
+      changes:
+        - dropForeignKeyConstraint:
+            baseTableName: recent_views
+            constraintName: fk_recent_views_ref_user_id
+      rollback:
+        - addForeignKeyConstraint:
+            baseTableName: recent_views
+            constraintName: fk_recent_views_ref_user_id
+            referencedTableName: core_user
+            baseColumnNames: user_id
+            referencedColumnNames: id
+            onDelete: CASCADE
+
+  - changeSet:
+      id: v48.00-048
+      author: noahmoss
+      comment: Add foreign key on recent_views with onDelete CASCADE
```

Comment:
Did this to fix a failing test which couldn't delete a user because it had entries in `recent_views`. It's not strictly necessary in the app itself though since users are never deleted, just deactivated.